addons/default/visiosoft/advs-module/resources/views/new-ad/new-create.twig

```diff
@@ -126,9 +126,20 @@
                                                value="{{ PriceValue[1] }}">
                                     </div>
                                     <div class="col-sm-3 col-xs-6">
-                                        {{ form.fields.currency.setAttributes({
-                                            'required' :true,
-                                        }).input|raw }}
+                                        <select name="currency" class="custom-select form-control" data-field="currency"
+                                                data-field_name="currency" data-provides="anomaly.field_type.select"
+                                                id="currency" required="">
+                                            {% for currency in config_get('streams::currencies.enabled') %}
+                                                <option value="{{ currency }}">
+                                                    {% if currency == "TRY" %}
+                                                        {{ currency_symbol(currency) }}
```

Comment:
buna genel geçer bir çözüm bulmamız lazım yine pyro ile kısıtlıyoruz kendimizi @spektra2147

Reply:
pyrodaki simgeleri düzenleyebiliriz. ama ozaman ilan fiyat yerinde $ yerine USD yazar.

yani sembol sembolü getirir isim isimi.

Reply:
I agree with Fatih

Reply:
we need to merge it beacuse it's fix. Need to create new issue for centrelize it.  @spektra2147 @ozcan-durak